README.md

```diff
@@ -1,12 +1,24 @@
 # Programming with Python
+
 [![GitHub release](https://img.shields.io/github/release/swcarpentry/python-novice-inflammation.svg)](https://github.com/swcarpentry/python-novice-inflammation/releases)
 [![Create a Slack Account with us](https://img.shields.io/badge/Create_Slack_Account-The_Carpentries-071159.svg)](https://swc-slack-invite.herokuapp.com/)
 [![Slack Status](https://img.shields.io/badge/Slack_Channel-swc--py--inflammation-E01563.svg)](https://swcarpentry.slack.com/messages/C9Y0L6MF0)
 
+
+## About the Lesson
+
 An introduction to Python for non-programmers using inflammation data.
 
+This lesson teaches novice programmers to write modular code to perform data analysis using Python. However, the emphasis is on teaching language-agnostic principles such as automation with loops and encapsulation with functions (see [Good enough practices in scientific computing][good-enough-practices] to
+learn more).
+
+The example used in this lesson analyses a set of 12 data files with inflammation data collected from a trial for a new treatment for arthritis (the data was simulated). Learners are shown how it is better to create a function and apply it to each of the 12 files using a loop instead of using copy-paste to analyse the 12 files individually.
```

Comment:
`the data was simulated` ~> `simulated data`

Reply:
I think the last sentence could be improved. Don't mention `a loop`. Use phrases like `Python enables ...` and `automation` when talking about Python, and , `error-prone` (or similar) when talking about analysis done "by hand".